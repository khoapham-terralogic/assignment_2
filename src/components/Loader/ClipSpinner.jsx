import React from "react";
import ClipLoader from "react-spinners/ClipLoader";

// Can be a string as well. Need to ensure each key-value pair ends with ;

export const ClipSpinner = ({ css }) => {
    return (
        <div>
            <ClipLoader
                css={css}
                size={50}
                color={"#4d0c5a"}
            />
        </div>
    )
}
