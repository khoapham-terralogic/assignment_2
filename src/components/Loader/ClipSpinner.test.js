import React from 'react'
import { shallow } from 'enzyme'
import { ClipSpinner } from './ClipSpinner'

describe("<ClipSpinner />", () => {
    test("ClipSpinner rendered", () => {
        const component = shallow(<ClipSpinner size={15} color="#fff" />)
        expect(component).toMatchSnapshot();
    })
})
