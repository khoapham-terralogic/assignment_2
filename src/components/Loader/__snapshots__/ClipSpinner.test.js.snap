// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ClipSpinner /> ClipSpinner rendered 1`] = `
<div>
  <Loader
    color="#fff"
    css=""
    loading={true}
    size={15}
  />
</div>
`;
